bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -292,9 +292,19 @@ impl Script {
         instructions.next().is_none()
     }
 
-    /// Checks whether a script pubkey is a Segregated Witness (segwit) program.
-    #[inline]
+    /// Checks whether a script pubkey is a Segregated Witness (segwit) script pubkey.
+    ///
+    /// This is being renamed to `is_segwit_script_pubkey` to be consistent with
+    /// [`SegwitScriptPubkey`](super::segwit::SegwitScriptPubkey) type which was previously
+    /// incorrectly named.
+    #[deprecated = "the name was misleading, use `is_segwit_script_pubkey` instead"]
```

Comment:
In b4544017caf0de075813b619607a9119bf78eaa2:

Can you add a `since = "TBD"` here (and in one other place).